pages/osx/pbcopy.md

```diff
@@ -1,6 +1,6 @@
 # pbcopy
 
-> Place standard output in the clipboard.
+> Copy data from STDIN to the clipboard.
```

Comment:
Good catch! One small nitpick; at present most of the `tldr` pages prefer `stdin` over `STDIN`. Can we lower case this description to conform to that pattern?

```
❯ rg 'STDIN' -l | wc -l
       4
❯ rg 'stdin' -l | wc -l
     164
```

```suggestion
> Copy data from stdin to the clipboard.
```